logger/logger.go

```diff
@@ -6,14 +6,9 @@ import (
 	"strings"
 	"time"
 
-	std_log "log"
-
-	go_log "git.sr.ht/~spc/go-log"
+	logrus "github.com/sirupsen/logrus"
 )
 
-const defaultLogFormat = 0
-const defaultLogPrefix = ""
-
 const (
```

Comment:
Can we remove the custom log levels or maybe alias the levels from logrus?
```go
const (
	DebugLevel = logrus.DebugLevel
	InfoLevel  = logrus.InfoLevel
	WarnLevel  = logrus.WarnLevel
	ErrorLevel = logrus.ErrorLevel
	TraceLevel = logrus.TraceLevel
)
```

Reply:
Thanks for highlighting this, will make the changes :)